src/lib/enums/SubmodelElementSemanticId.enum.ts

```diff
@@ -29,4 +29,5 @@ export enum SubmodelElementSemanticId {
     EntryNode = 'https://admin-shell.io/idta/HierarchicalStructures/EntryNode/1/0',
     BulkCount = 'https://admin-shell.io/idta/HierarchicalStructures/BulkCount/1/0',
     ArcheType = 'https://admin-shell.io/idta/HierarchicalStructures/ArcheType/1/0',
+    SerialNumber = '0173-1#02-AAM556#002',
```

Comment:
Maybe name it "NameplateSerialNumber" to make clear which submodel it belongs to

Reply:
I delete it, because we only introduced it for testing :).